.github/workflows/tests.yml

```diff
@@ -53,3 +53,20 @@ jobs:
         with:
           args: "--validate"
         if: matrix.python-version == '3.8'
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
```

Comment:
Let's use a string literal instead of integer literal for consistency: `node-version: "16"`.